flowauth/frontend/src/Dashboard.js

```diff
@@ -171,6 +171,7 @@ class Dashboard extends React.Component {
               classes.appBar,
               this.state.open && classes.appBarShift
             )}
+            color={is_admin ? "secondary" : "primary"}
```

Comment:
I trust this works, but where are the "primary" and "secondary" colours actually set? They don't seem to be mentioned elsewhere in this file, nor in any of the css files?

Reply:
They come from the material-ui package.

Reply:
I see, thanks! Not sure if it's worth adding a comment to this effect or if this is effectively common knowledge. Out of interest, are we using a default palette or defining our own somewhere?

Reply:
Default palette, but we'll need to have a branding exercise at some point I think.